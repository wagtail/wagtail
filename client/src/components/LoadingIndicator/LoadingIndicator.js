import React from 'react';
import { STRINGS } from '../../config/wagtail';

const LoadingIndicator = () => (
  <div className="o-icon c-indicator is-spinning">
    <span ariaRole="presentation">{STRINGS.LOADING}</span>
  </div>
);

export default LoadingIndicator;
