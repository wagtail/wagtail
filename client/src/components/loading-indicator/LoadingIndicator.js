import React from 'react';

const LoadingIndicator = () => (
    <div className="o-icon c-indicator is-spinning">
        <span ariaRole="presentation">Loading...</span>
    </div>
);

export default LoadingIndicator;
