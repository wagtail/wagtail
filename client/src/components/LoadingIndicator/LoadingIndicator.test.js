import React from 'react';
import { shallow } from 'enzyme';

import LoadingIndicator from './LoadingIndicator';

describe('LoadingIndicator', () => {
  it('exists', () => {
    expect(LoadingIndicator).toBeDefined();
  });

  it('basic', () => {
    expect(shallow(<LoadingIndicator />)).toMatchSnapshot();
  });
});
