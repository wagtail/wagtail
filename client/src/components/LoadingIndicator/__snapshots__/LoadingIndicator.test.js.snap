exports[`LoadingIndicator basic 1`] = `
<div
  className="o-icon c-indicator is-spinning">
  <span
    ariaRole="presentation">
    Loading...
  </span>
</div>
`;
